demosymfonyforms/src/Uploader/SupplierSecondImageUploader.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
```

Comment:
License must be AFL, not OSL. See modules like https://github.com/friends-of-presta/demo-cqrs-hooks-usage-module